metabase/metabase
# Migrate GoogleSettingsForm to formik and add unit tests

Epic https://github.com/metabase/metabase/issues/26178

How to test:
- Admin -> Settings -> Authentication -> Google
- Use `fake.apps.googleusercontent.com` for the `Client ID` and make sure it is possible to submit the form

<img width="651" alt="Screenshot 2022-11-08 at 18 08 21" src="https://user-images.githubusercontent.com/8542534/200616289-40dfb7ec-00e1-4603-8a0e-4ce865846cce.png">
